classes/rest/class-rest-api.php

```diff
@@ -252,6 +253,26 @@ public static function endpoints(): void {
 				],
 			]
 		);
+
+		/**
+		 * Endpoint to retrieve the covers for the Covers block
+		 */
+		register_rest_route(
+			self::REST_NAMESPACE,
+			'/get-covers',
+			[
+				[
+					'permission_callback' => static function () {
+						return true;
+					},
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => static function ( $fields ) {
+						$covers = Covers::get_covers( $fields );
+						return rest_ensure_response( $covers );
```

Comment:
Can you set this data the same way we did for the Gallery block? https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/45ab1881ae40ccb5e88ae62c8ca9c7a6e63ef6e6/classes/blocks/class-gallery.php#L46-L52

Reply:
Done!

Reply:
Nice. We should probably also use `useSelect` in the editor like we did there. Then we can remove this endpoint and can probably simplify the block's PHP functions too.
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/2e695d0ed1e4db9755c554666c6044c97ab56bb1/assets/src/blocks/Gallery/GalleryEditor.js#L160-L172

Reply:
It would also be faster in most cases, since `useSelect` will cache things, while our endpoint will keep getting called.

Reply:
This will be taken care of in a [follow-up ticket](https://jira.greenpeace.org/browse/PLANET-5975)!